db/migrate/20220901092755_create_client_testimonials.rb

```diff
@@ -0,0 +1,11 @@
+class CreateClientTestimonials < ActiveRecord::Migration[7.0]
+  def change
+    create_table :client_testimonials do |t|
+      t.string :testimonial, default:"-"
+      t.string :author_name, default: "-"
```

Comment:
Give proper naming for author_name and author_designation

Reply:
The fields are for the author of testimonial, the client author belongs to is a reference key. So the names seem correct to me😅

Reply:
Auhtor of testimonial is known as attestant. So it will be better if you change author_name to attestant and author_designation to attestant_designation. Give a thought over it